test/test_transactions_lasse.cpp

```diff
@@ -102,19 +95,11 @@ REALM_FORCEINLINE void rand_sleep(Random& random)
     }
     else if (r <= 254) {
 // Release current time slice and get time slice according to normal scheduling
-#ifdef _MSC_VER
-        Sleep(0);
-#else
-        usleep(0);
-#endif
+        millisleep(0);
```

Comment:
I think that in this particular unit test, it doesn't matter. Note that in other places you may need to clealy distinguish between a 0-delay in the same OS time slice and a yield